README.md

```diff
@@ -133,6 +133,20 @@ We are always trying to improve our documentation, if you have suggestions or ru
 
 ##### Likely only useful for 18F team members
 
+##### Create change log
+
+Create a change log for each release. Auto generate it using this [change log generatorr](https://github.com/skywinder/github-changelog-generator)
+
+Once installed, you will want to run:
+
+```
+github_changelog_generator --since-tag <last public-relase> --t <your-gh-token>
```

Comment:
Maybe, I am not understanding your question, but the change log and deploy are separate things to me. This part is just instructions on how to make a change log. The deploy steps are in the next section.